integration_tests/broker_test.go

```diff
@@ -87,13 +158,16 @@ var _ = Describe("Broker", func() {
 			uriParam{key: "plan_id", value: planID},
 			uriParam{key: "accepts_incomplete", value: strconv.FormatBool(acceptsIncomplete)},
 		)
-		resp := serveRequest(responseRecorder, req)
-		Expect(responseRecorder.Code).To(Equal(http.StatusInternalServerError))
-		body := readResponseBody(resp)
-		Expect(string(body)).To(ContainSubstring("Can't deprovision an instance"))
+		brokerAPI.ServeHTTP(responseRecorder, req)
+
+		foundDeployments := *fakeComposeClient.FoundDeployments
+		Expect(fakeComposeClient.DeprovisionDeploymentID).To(Equal(foundDeployments[1].ID))
```

Comment:
Addressed by 5691b63295918cf5fe12e06989aa60edb365a2e3 which can be squashed when Piotr is happy with it.